src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-860.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2022-04-04'
 version: 8.6.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.6.0.gem'
+features: ["Telemetry-in-Context: Automatic Application Logs, a quick way to view logs no matter where you are in the platform"]
```

Comment:
I wonder if we should use "APM Logs in Context" here, since the verbiage seems to reference the project using this phrase now instead of "Telemetry-in-Context"?

Could be something like:
"Introduces APM Automatic logs-in-context, a feature to forward application logs directly to New Relic"